docs/azure_arc_jumpstart/azure_arc_data/gke/gke_dc_vanilla_terraform/_index.md

```diff
@@ -45,43 +45,36 @@ By the end of this scenario, you will have a GKE cluster deployed with an Azure
 
 * Google Cloud account with billing enabled - [Create a free trial account](https://cloud.google.com/free). To create Windows Server virtual machines, you must upgraded your account to enable billing. Click Billing from the menu and then select Upgrade in the lower right.
 
-    ![Screenshot showing how to enable billing on GCP account](./44.png)
+    ![Screenshot showing how to enable billing on GCP account](./02.png)
 
-    ![Screenshot showing how to enable billing on GCP account](./45.png)
+    ![Screenshot showing how to enable billing on GCP account](./03.png)
 
-    ![Screenshot showing how to enable billing on GCP account](./46.png)
+    ![Screenshot showing how to enable billing on GCP account](./04.png)
 
     ***Disclaimer*** - **To prevent unexpected charges, please follow the "Delete the deployment" section at the end of this README**
 
 * [Install Terraform 1.0 or higher](https://learn.hashicorp.com/terraform/getting-started/install.html)
 
-* Create Azure service principal (SP). To deploy this scenario, an Azure service principal assigned with multiple RBAC roles is required:
+* Create Azure service principal (SP). To deploy this scenario, an Azure service principal assigned with a RBAC role is required:
 
-  * "Contributor" - Required for provisioning Azure resources
-  * "Security admin" - Required for installing Cloud Defender Azure-Arc enabled Kubernetes extension and dismiss alerts
-  * "Security reader" - Required for being able to view Azure-Arc enabled Kubernetes Cloud Defender extension findings
-  * "Monitoring Metrics Publisher" - Required for being Azure Arc-enabled data services billing, monitoring metrics, and logs management
+  * "Owner" - Required for provisioning Azure resources, interact with Azure Arc-enabled data services billing, monitoring metrics, and logs management and creating role assignment for the Monitoring Metrics Publisher role.
 
     To create it login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/).
```

Comment:
```suggestion
    To create it login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
```